src/client/cosmos.ts

```diff
@@ -45,25 +46,11 @@ export default class CosmosDbClient {
     account: string,
```

Comment:
it looks like this method doesn't need to be exported any more. makes sense to make it `private`?